src/webpack/webpackTools.ts

```diff
@@ -27,17 +28,27 @@ export interface CustomWebpackOptions extends webpack.Options.Optimization {
      * Allows a custom output file name to be used instead of the default [name]-bundle.js
      */
     outputFilename?: string;
+
+    /**
+     * Allows CSS to be extracted into a seperate file. If this is not specified, CSS will be bundled in the javascript bundle.
+     */
+    extractStyles?: boolean | string;
 }
 
 /**
  * Shared function to create a webpack config for an entry point
  */
-export function createConfig(outPath: string, entryPoint: string | webpack.Entry, options: CustomWebpackOptions, ...plugins: webpack.Plugin[]) {
+
+export function createConfig(outPath: string, entryPoint: string | webpack.Entry, options?: CustomWebpackOptions, ...plugins: webpack.Plugin[]) {
+    let extractCSS = false;
+    if (options && options.extractStyles) {
+        extractCSS = !!options.extractStyles || false;
+    }
```

Comment:
Seconding Ryan's comment about making this one line so `extractCSS` can be `const`. Also not sure `|| false` will do anything in this case, given we've already got the `!!` cast to boolean